From 587b2270b87aada358ae5ddd4e44e9ca1a0eae05 Mon Sep 17 00:00:00 2001
From: Jeff King <peff@peff.net>
Date: Tue, 18 Oct 2011 20:12:58 -0400
Subject: [PATCH] pack-objects: drop "pack not accessible" warning entirely

This is a GitHub-only patch. In most instances, this warning
might be something you want to know about (e.g., somebody is
pruning behind your back, or there really is an error with
one of your packs). But GitHub prunes very aggressively,
which means this actually comes up reasonably frequently if
you have a busy repo.

Without this patch, fetching users may see:

  remote: warning: packfile pack-cbec5abc4e528d54ed4d64295793579bff4d3e28.pack cannot be accessed

even though it is totally uninteresting to them, and not
indicative of a problem at all. So let's make sure they
don't ever see it.
---
 builtin/pack-objects.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/builtin/pack-objects.c b/builtin/pack-objects.c
index 7b07c09..d3e5091 100644
--- a/builtin/pack-objects.c
+++ b/builtin/pack-objects.c
@@ -845,7 +845,6 @@ static int add_object_entry(const unsigned char *sha1, enum object_type type,
 		if (offset) {
 			if (!found_pack) {
 				if (!is_pack_valid(p)) {
-					warning("packfile %s cannot be accessed", p->pack_name);
 					continue;
 				}
 				found_offset = offset;
-- 
1.7.9.6.6.g6b3b56

